.changeset/green-donuts-lick.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+fix: integration tests fail silently
```

Comment:
This should go after https://github.com/FuelLabs/fuels-ts/pull/2991 as it could block the release by making the integration test stage fail correctly.

Reply:
Good to go. 🫡